scout/server/utils.py

```diff
@@ -205,7 +205,26 @@ def user_institutes(store, login_user):
     return institutes
 
 
-def case_has_alignments(case_obj):
+def case_has_chanjo_coverage(case_obj: dict):
+    """Return True if case has coverage stats in chanjo."""
+
+    chanjo_instance: bool = bool(current_app.config.get("SQLALCHEMY_DATABASE_URI"))
+    if case_obj.get("track", "rare") != "cancer" and chanjo_instance:
+        case_obj["chanjo_coverage"] = True
+
+
+def case_has_chanjo2_coverage(case_obj: dict):
+    """Return True if case has coverage stats in chanjo2."""
+
+    chanjo2_instance: bool = bool(current_app.config.get("CHANJO2_URL"))
+    for ind in case_obj.get("individuals", []):
+        ind_d4: str = ind.get("d4_file")
+        if ind_d4 and os.path.exists(ind_d4) and chanjo2_instance:
```

Comment:
You could return false if not `chanjo2_instance` earlier and avoid this loop and extra test?

Reply:
I am not actually returning anything in this function and the one before, I am setting key/values for the case. I think I had it different before. I'll just fix the docstring

Reply:
I want to set case_obj["chanjo2_coverage"] to True is at least one of the samples have an associated d4 file

Reply:
> You could return false if not `chanjo2_instance` earlier and avoid this loop and extra test?

Ah but you are right, I could simply return

Reply:
Fixed as you suggested, thanks!